### DISCUSSION TITLE
Add infos to doc how to add PhpStorm protocol for Windows.

### DISCUSSION THREAD

**Discussion by oliverhermanni** [spatie/ray#118] [Ideas]
Files in the Ray app won't open, because Windows doesn't understand the PhpStorm protocol. This issue can be fixed by using [this](https://github.com/aik099/PhpStormProtocol). It might sound a little bit fishy to add something to the registry by hand, but the solution is also mentioned on [Laracasts](https://laracasts.com/discuss/channels/general-discussion/cant-get-ignition-to-open-file-in-phpstorm).

I recommend to add this info to the docs.

**Comment by freekmurze** [comment]
Thanks for mentioning this. Could you PR this to the docs? There's an edit button above every page. This way you get contribution credits 🙂

**Reply by oliverhermanni** [reply]
Yay!
